openassessment/xblock/static/js/spec/oa_server.js

```diff
@@ -56,6 +56,43 @@ describe("OpenAssessment.Server", function() {
         '</criterion>'+
     '</rubric>';
 
+    var RUBRIC_JSON = {
+        "criteria": [
+            {
+                "label": "𝓒𝓸𝓷𝓬𝓲𝓼𝓮",
```

Comment:
dude... wat?

Reply:
Lol, this was an existing example, so I copied it. Not sure what cursed back room they got those characters from but I'm about it.